src/data/set/basic.lean

```diff
@@ -1356,6 +1359,15 @@ by { split_ifs with hb hb, exacts [preimage_const_of_mem hb, preimage_const_of_n
 
 theorem preimage_comp {s : set γ} : (g ∘ f) ⁻¹' s = f ⁻¹' (g ⁻¹' s) := rfl
 
+lemma preimage_comp' : preimage (g ∘ f) = preimage f ∘ preimage g := rfl
```

Comment:
If we follow the convention you're trying to establish in this file, shouldn't this rather be `preimage_comp_eq`?

Reply:
OK sure.